Weather App/controller/SearchTabViewController.swift

```diff
@@ -0,0 +1,89 @@
+import SnapKit
+import UIKit
+
+class SearchTabViewController: UIViewController {
+    private var searchTextField: UISearchBar!
+    private var tableView: UITableView!
+    private let cities = ["Łódź", "Warszawa", "Kraków", "Wrocław", "Gdańsk", "Gdynia", "Bydgoszcz", "Sopot", "Kielce"]
+    private var dataToDispaly: [String]!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .systemCyan
+
+        dataToDispaly = cities
+
+        configureTextField()
+        configureTable()
+    }
+}
+
+extension SearchTabViewController {
+    private func configureTextField() {
+        searchTextField = UISearchBar()
+        searchTextField.barTintColor = .systemCyan
+        searchTextField.layer.cornerRadius = 15.0
+        searchTextField.placeholder = "Search City"
+        searchTextField.searchBarStyle = .minimal
+        searchTextField.delegate = self
+
+        view.addSubview(searchTextField)
+
+        searchTextField.snp.makeConstraints { make in
+            make.height.equalTo(60)
+            make.leading.trailing.equalToSuperview().inset(16)
+            make.top.equalTo(view.snp_topMargin)
+        }
+    }
+
+    private func configureTable() {
+        tableView = UITableView(frame: .zero, style: .insetGrouped)
+        tableView.backgroundColor = .systemCyan
+        tableView.dataSource = self
+        tableView.delegate = self
+
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "cell")
```

Comment:
I recommend you store names of cells identifiers in some `Constants` struct inside class. It will be more elegant and it easier to change in one place. It also be good place to store names, colors, constraints, etc 

`struct Constants {

        private init() { }

        static let hoursSelectionCellIdentifier = "searchCell"

        // MARK: - CollectionView's Constants
        static let collectionViewLeading: CGFloat = 24
        static let collectionViewTrailing: CGFloat = 24
        
        // MARK: - ButtonStackView's Constants

        static let buttonStackViewSpacing: CGFloat = 8
    }

    struct Texts {

        private init() { }

        static let text = "text"
    }`

Reply:
Added